sphinxcontrib/openapi.py

```diff
@@ -138,6 +155,7 @@ class OpenApi(Directive):
     final_argument_whitespace = True        # path may contain whitespaces
     option_spec = {
         'encoding': directives.encoding,    # useful for non-ascii cases :)
+        'paths': directives.unchanged_required,
     }
 
```

Comment:
Take a look at the new implementation I just pushed, it uses the `option_spec` mapping as intended, and uses the _clean_paths_option` transformation function directly. I misread the Directive docs when I first implemented this feature.